pages/benchmarks.tsx

```diff
@@ -173,6 +187,32 @@ const Benchmarks = () => {
                 </p>
               </div>
             </div>
+            <div className="mt-20">
+              <h4 className="text-2xl font-bold tracking-tight">
+                TypeScript Performance
+              </h4>
+              <div className="mt-8">
+                <h5 className="text-lg font-medium tracking-tight">
+                  Type Checking
+                </h5>
+                <BenchmarkOrLoading
+                  data={data}
+                  columns={data?.execTime.filter(({ name }) => {
+                    console.log(name);
+                    return ["check", "no_check"].includes(name);
+                  })}
+                  yLabel="seconds"
+                />
+                <p className="mt-1">
+                  In both cases, a full type check is run for&nbsp;
```

Comment:
Why are all of the non breaking spaces needed?

Reply:
Because with the formatting, you get things like "Withcheck a" when using things like `<em>`.

Reply:
Got it - you can replace all of the &nbsp; with `{" "}`. It serves the same purpose and prettier should intelligently add/remove these when needed. Non breaking spaces have the unintended side effect that they are non breaking :-)